frontendcomponent/androidcomponent/src/main/res/layout/section.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="vertical"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content">
+    <TextView
+        android:id="@+id/section_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/gray1"
+        android:textSize="24sp"
+        android:text="@string/session_label"
+        android:layout_marginTop="22dp"
```

Comment:
Here as well I think should be `24dp` or `28dp` (it looks different between `Speaker` and `Sessions`, but maybe we can simplify and make them the same)

<img width="435" alt="screen shot 2019-01-22 at 22 44 18" src="https://user-images.githubusercontent.com/1574446/51539519-96c68b80-1e97-11e9-9de4-5a5e4360ec27.png">

Reply:
I will change 28dp!